tcms/testruns/templates/testruns/get.html

```diff
@@ -63,41 +63,44 @@ <h2 class="card-pf-title" style="text-align: left">
                     </h2>
                 {% endif %}
 
-                <h2 class="card-pf-title" style="text-align: left">
-                    <span class="fa fa-spinner"></span>{% trans 'Status' %}:
-                    <input class="bootstrap-switch" id="status_button" type="checkbox"
-                        {% if not object.stop_date %}checked{% endif %}
-                        {% if not perms.testruns.change_testrun %}disabled{% endif %}
-                        data-on-text="{% trans 'Running' %}" data-off-text="{% trans 'Finished' %}">
-                </h2>
 
                 <h2 class="card-pf-title" style="text-align: left">
                     <span class="fa fa fa-calendar-o"></span>{% trans 'Planned start' %}:
                     {{ object.planned_start|default:'-'}}
                 </h2>
 
-                <h2 class="card-pf-title" style="text-align: left">
-                    <span class="fa fa-calendar"></span>{% trans 'Started at' %}:
-                    {{ object.start_date }}
+                <h2 class="card-pf-title" style="text-align: left;">
+                    <span class="fa fa-calendar"></span>
+                        {% trans 'Started at' %}:
+                    <span class="start-date">
+                            {{ object.start_date|default:'-' }}
+                        </span>
+                    {% if not object.start_date %}
+                        <button class="btn btn-success btn-xs" type="button" id="start-button"
+                                data-toggle="tooltip" data-placement="top" title="{% trans 'Start' %}"
+                        style="float:right;">
```

Comment:
can you either 
1) align the attribites by column so they are more readable or
2) move this at the end of previous line like you have on L98-99.